addons/website_sale/static/src/xml/website_sale.editor.xml

```diff
@@ -60,4 +60,12 @@
     </div>
 </div>
 
+<div t-name="website_sale.dialog_confirmation">
+    Are you sure that you want to change the <em>size, sequence, number of products or column</em>? After confirming you can not discard the changes.
```

Comment:
"Are you sure you want to modify the shop grid ?" is clearer and more generic to me